README.md

```diff
@@ -144,10 +175,29 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ### 2 or more PRs
 
+#### **Accord Project** (stickers)
+
+- **Requirements**: 2 pull requests across any difficulty
+- **Swag**: limited edition stickers
+- **How to sign up**: Make pull requests on the participating repositories and respond to us when asked for shipping details.
+- **Notes**: Check out the [official page](https://www.accordproject.org/events/hacktoberfest-2019/) for more information.
+
+#### **Accord Project** (limited edition t-shirt and stickers)
+
+- **Requirements**: either of the three
+    * 1 valid `Difficulty: Challenging` PR
+    * 2 valid `Difficulty: Medium` PR
+    * 5 valid `Difficulty: Starter` PR
```

Comment:
Please remove the ` symbols here as they are used for inline code. You can see how it affects the Jekyll build by [looking at the deploy preview here](https://deploy-preview-86--hacktoberfest-swag-list.netlify.com/)